src/hw-4/models/userGroup.model.ts

```diff
@@ -0,0 +1,40 @@
+import { DataTypes, Sequelize } from 'sequelize';
+import dbConnection from '../services/db.service';
+
+import GroupModel from './group.model';
+import UserModel from './user.model';
+
+const UserGroupModel = (sequelize: Sequelize) => {
```

Comment:
be honest I don't understand why you need this UserGroup model when ORM does this linking between two models in similar way under the hood.